### PR TITLE
docs for pr https://github.com/rundeck/rundeck/pull/5577

### PR DESCRIPTION
**Pull request by ronaveva**
docs for pr https://github.com/rundeck/rundeck/pull/5577

### CODE REVIEW

File: docs/administration/projects/node-execution/ssh.md [hunk 1]
```diff
@@ -571,17 +571,37 @@ environment.
...
     AcceptEnv RD_*
 
 [ssh]: https://en.wikipedia.org/wiki/Secure_Shell
+
+
+- To pass environment variables directly throught the command it self, you 
+can also make use of these node attributes to stablish the configuration by
+node
+  - ssh-variable-export-pattern : [STRING] Allows to explictly set the export 
+  sintaxis.
```
Comment (gschueler) [inline docs/administration/projects/node-execution/ssh.md]:
```suggestion
  pattern.
```

Comment (gschueler) [reply docs/administration/projects/node-execution/ssh.md]: also, please add info about `{key}` and `{value}`

**Review by gschueler — CHANGES_REQUESTED** [review]

File: docs/administration/projects/node-execution/ssh.md (hunk 1 above)
Comment (ronaveva) [reply docs/administration/projects/node-execution/ssh.md]: > 
> 
> also, please add info about `{key}` and `{value}`

Added

File: docs/administration/projects/node-execution/ssh.md [hunk 2]
```diff
@@ -571,17 +571,43 @@ environment.
...
     AcceptEnv RD_*
 
 [ssh]: https://en.wikipedia.org/wiki/Secure_Shell
+
+
+- To pass environment variables directly throught the command it self, you 
+can also make use of these node attributes to stablish the configuration by
+node
+  - ssh-variable-export-pattern : [STRING] Allows to explictly set the export 
+  sintaxis, it must have {key} and {value}.
```
Comment (gschueler) [inline docs/administration/projects/node-execution/ssh.md]: `sintaxis` is not correct english, either "syntaxes", or "pattern", i think pattern is better
```suggestion
  pattern, it must have {key} and {value}.
```

**Review by gschueler — CHANGES_REQUESTED** [review]

**Pull request merged**